AssetRipper.GUI/ConsoleOptions.cs

```diff
@@ -0,0 +1,350 @@
+						{
+							symbolResult.ErrorMessage = MultiFileStream.IsMultiFile(path)
+								? $"File '{path}' doesn't have all parts for combining"
+								: $"Neither file nor directory with path '{path}' exists";
+
+							Environment.ExitCode = (int)ExitCode.ArgumentError;
+							return;
+						}
+					}
+				});
+
+				_inputArgument = argument;
+
+				return argument;
+			}
+		}
+
+		private static Option<List<string>>? _inputOption;
+
+		public static Option<List<string>> inputOption
+		{
+			get
+			{
+				if (_inputOption != null)
+					return _inputOption;
+
+				Option<List<string>> option = new Option<List<string>>(name: "--input", description: "Input files or directory to export");
+				option.AddAlias("-i");
+				option.IsRequired = true;
+				option.Arity = ArgumentArity.OneOrMore;
+
+				option.AddValidator((symbolResult) =>
+				{
+					List<string>? inputs = symbolResult.GetValueForOption(option);
+
+					if (inputs == null)
+						return;
+
+					foreach (string input in inputs)
+					{
+						string path = ExecutingDirectory.Combine(input);
+
+						if (!MultiFileStream.Exists(path) && !Directory.Exists(path))
+						{
+							symbolResult.ErrorMessage = MultiFileStream.IsMultiFile(path)
+								? $"File '{path}' doesn't have all parts for combining"
+								: $"Neither file nor directory with path '{path}' exists";
+
+							Environment.ExitCode = (int)ExitCode.ArgumentError;
+							return;
+						}
+					}
+				});
+
+				_inputOption = option;
+
+				return option;
+			}
+		}
+
+		private static Option<DirectoryInfo>? _outputOption;
+
+		public static Option<DirectoryInfo> outputOption
+		{
+			get
+			{
+				if (_outputOption != null)
+					return _outputOption;
+
+				Option<DirectoryInfo> option = new Option<DirectoryInfo>(name: "--output", description: "Directory to export to (will be cleared if already exists)");
+				option.AddAlias("-o");
+				option.SetDefaultValue(DefaultOutputPath);
+				option.IsRequired = false;
+
+				option.AddValidator((symbolResult) =>
+				{
+					DirectoryInfo? dirInfo = symbolResult.GetValueForOption(option);
+
+					if (dirInfo == null)
+						dirInfo = new DirectoryInfo(DefaultOutputPath);
+
+					string path = ExecutingDirectory.Combine(dirInfo.FullName);
+
+					if (!Directory.Exists(Path.GetDirectoryName(path)))
+					{
+						symbolResult.ErrorMessage = "Directory for output does not exist";
+						Environment.ExitCode = (int)ExitCode.ArgumentError;
+					}
+				});
+
+				_outputOption = option;
+
+				return option;
+			}
+		}
+
+		private static Option<bool>? _quitOption;
+
+		public static Option<bool> quitOption
+		{
+			get
+			{
+				if (_quitOption != null)
+					return _quitOption;
+
+				Option<bool> option = new Option<bool>(name: "--quit", description: "Quit after ripping");
+				option.AddAlias("-q");
+				option.SetDefaultValue(DefaultQuit);
+
+				_quitOption = option;
+
+				return option;
+			}
+		}
```

Comment:
I thought we agreed on getting rid of this.

Reply:
Yep, I actually removed it first but added it back as it was originally to not break existing scripts. Someone may be using AssetRipper in a script already and upgrading to this would break those use cases. Since this pr is just about adding unavailable parameters, I think it's best to consider removing that in the future.